### PR TITLE
Selectbox checkbox option

### PR DESCRIPTION
**Pull request by rokburgar**
@gregorpraprotnik Can you check this option?

### CODE REVIEW

File: src/selectbox_abstract.coffee [hunk 1]
```diff
@@ -164,6 +166,16 @@ class SelectBoxAbstract
...
 
+            if @options.showcheckbox and @orig.is '[multiple]'
+                checkboxOptions =
+                    type  : 'checkbox'
+                    class : 'reform-checkbox'
+
+                checkboxOptions.checked = 'checked'  if $option.is ':selected'
+                checkboxOptions.checked = 'disabled' if $option.is ':disabled'
+
+                $item.append($('<input>', checkboxOptions))
```
Comment (gregorpraprotnik) [inline src/selectbox_abstract.coffee]: I like this syntax :+1:

**Comment by gregorpraprotnik** [comment]
@rokburgar the code looks okay to me. However the UI could be improved. When you select the first item all items are shifted down to accommodate room for selected items label at the top. It's confusing when this happens. I suggest skip the selected items label and just put them in the input.

**Comment by rokburgar** [comment]
@gregorpraprotnik Well I didn't do this behavior in this branch... My only change was that you have an options to add checkboxes on multiple selectbox.

File: src/selectbox_abstract.coffee (hunk 1 above)
Comment (rokburgar) [inline src/selectbox_abstract.coffee]: @gregorpraprotnik Found one mistake should be .disabled = 'disabled'. Will change it tomorrow.

**Comment by gregorpraprotnik** [comment]
Must be some kind of a setting then? Because some combos in demo are working as yours and some as I described...

Please make sure that your changes look ok with other themes under the #themes branch,

**Comment by rokburgar** [comment]
@gregorpraprotnik  Well I guess if I add data-plural="numbers" it will display on input... Didn't you do this? :) Well anyway I don't see how this is related to this branch...

Tried it on other themes. Looks like it should.

**Comment by gregorpraprotnik** [comment]
@rokburgar Looks good to me then.

**Pull request merged**